### PR TITLE
Add Vorbis benchmark.

### PR DESCRIPTION
**Pull request by morehouse**
Resolves https://github.com/google/fuzzer-test-suite/issues/115.

### CODE REVIEW

File: vorbis-2017-12-11/build.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  (cd BUILD/vorbis && ./autogen.sh && ./configure \
+    --prefix="$INSTALL_DIR" \
+    --enable-static \
+    --disable-shared \
+    && make clean && make -j $JOBS && make install)
+}
+
+download_fuzz_target() {
+  [[ ! -e SRC/oss-fuzz ]] && \
+    git clone -n https://github.com/google/oss-fuzz.git SRC/oss-fuzz
```
Comment (kcc) [inline vorbis-2017-12-11/build.sh:34]: why not get_git_revision?

**Comment by kcc** [comment]
LGTM with a nit

**Comment by morehouse** [comment]
Can't see the nit.  Did you submit the review?

File: vorbis-2017-12-11/build.sh (hunk 1 above)
Comment (morehouse) [reply vorbis-2017-12-11/build.sh:34]: `get_git_revision` downloads all files in the repo.  I only wanted to download one specific file.

I could use get_git_revision for simplicity, at the cost of slightly more space used.

Comment (kcc) [reply vorbis-2017-12-11/build.sh:34]: ok

File: vorbis-2017-12-11/build.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```
Comment (nguyenduongtruongtoan) [inline vorbis-2017-12-11/build.sh]: <Head>git clone https://github.com/googlecodelabs/accelerated-mobile-pages-foundations.git<hehe>

File: vorbis-2017-12-11/build.sh [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+# Copyright 2017 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+. $(dirname $0)/../custom-build.sh $1 $2
+. $(dirname $0)/../common.sh
+
```
Comment (nguyenduongtruongtoan) [inline vorbis-2017-12-11/build.sh]: <head>git clone https://github.com/googlecodelabs/accelerated-mobile-pages-foundations.git<hehe>

**Comment by nguyenduongtruongtoan** [comment]
git clone https://github.com/googlecodelabs/accelerated-mobile-pages-foundations.git

**Pull request merged**